docs/iidm/exporter/iidm.md

```diff
@@ -11,8 +11,16 @@ For more information about the IIDM model, see [here](../model/index.md).
 
 IIDM networks can be serialized in XML files. The IIDM exporter generates files with a `*.xiidm` extension.
 
-The IIDM exporter has three exporting modes:
+The IIDM exporter has two exporting types:
+   - **Full export**: Exports the whole network with all its elements. 
+   
+   - **Incremental export**: Generates (by default) three files:
+        - `base-STATE.xiidm` file : contains only network elements having state attributes.
+        - `base-TOPO.xiidm` file : contains only network elements having topology attributes.
+        - `base-CONTROL.xiidm` file : contains only network elements having control attributes.
 
+
+The first type ,full export, has three exporting modes:
```

Comment:
space must be **after** the comma, not before: ·`The first type, full export, has etc.`